src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -8,33 +9,152 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class ImageHandler : ViewHandler<IImage, Image>
 	{
+		/// <inheritdoc/>
 		protected override Image CreatePlatformView() => new Image();
 
+		/// <inheritdoc/>
 		protected override void DisconnectHandler(Image platformView)
 		{
 			base.DisconnectHandler(platformView);
 			SourceLoader.Reset();
 		}
 
+		/// <inheritdoc/>
 		public override bool NeedsContainer =>
 			VirtualView?.Background != null ||
+			VirtualView?.Aspect == Aspect.AspectFill ||
 			base.NeedsContainer;
 
+		/// <inheritdoc/>
+		protected override void SetupContainer()
+		{
+			base.SetupContainer();
+
+			// VerticalAlignment only works when the child's Height is Auto
+			PlatformView.Height = double.NaN;
```

Comment:
What about horizontal alignment? Does that just work?

Reply:
Not sure. Copied it from here per @PureWeen 
https://github.com/dotnet/maui/blob/7d5aa8ca42896b4056c9315235250be5b7c151f4/src/Core/src/Handlers/Label/LabelHandler.Windows.cs#L15C3-L30